docs/source/using-discovery/bash.rst

```diff
@@ -0,0 +1,867 @@
+.. _bash:
+
+***********
+Using Bash
+***********
+.. figure:: /images/bash-logo.png
+   :class: with-border
+   :width: 300
+   :alt: Bash (Bourne Again SHell) logo.
+   :align: center
+
+   Bash (Bourne Again SHell)
+
+
+Overview
+=========
+Bash (Bourne Again SHell) is a popular shell and command-line interface. Specifically, a shell is an interface between the user and the underlying operating system, allowing users to interact with the system and perform tasks. Bash provides a range of features for running commands, managing files, navigating systems, and performing other tasks.
+
+Bash commands perform various tasks within the shell environment. Commands span from basic functionalities (e.g., ``ls``, ``cd``, ``cp``, ``mv``, and ``rm``) to more advanced ones(e.g., ``grep`` and ``awk``). We cover these commands and more in this tutorial. Bash can also be used in scripts, allowing users to automate tasks and perform more complex operations via loops, conditional logic, and defining functions, which we cover at the end of this page.
+
+In summary, shell commands perform various tasks with the terminal.
+
+
+.. figure:: /images/terminal-view.png
+   :class: with-border
+   :width: 300
+   :alt: Terminal View.
+   :align: center
+
+   Terminal View
+
+Terminal
+=========
+The terminal - aka the command line interface (CLI) - is a text-based interface for interacting with an operating system. It is a way for users to interact with the system and perform tasks by typing commands and receiving text-based output.
```

Comment:
I would spell out aka - also know as - because someone might not be familiar with the abbreviation.

Reply:
You might what to rephrase the end of this statement because you don't always get a text-based output. cp, mv, and other commands don't give a confirmation after they perform their action. So you might want to say 'typing commands to perform actions' or something like that just because of the commands that don't output text.